.github/workflows/continuous-integration.yml

```
@@ -9,5 +9,5 @@ jobs:
   ci:
     uses: ray-di/.github/.github/workflows/continuous-integration.yml@v1
     with:
-      old_stable: '["8.1", "8.2", "8.3"]
-      current_stable: 8.4
+      old_stable: '["8.2", "8.3"]
```

actionlint (1.7.10)

could not parse as YAML: found unexpected end of stream

(syntax-check)